gradle.properties

```diff
@@ -3,7 +3,7 @@ group=io.ballerina
 version=0.2.1-SNAPSHOT
```

Comment:
Should we bump the version to 0.3.0-SNAPSHOT? @niveathika

Reply:
@HasithaAthukorala

Reply:
Ack, I'll fix this in a separate PR